packages/experiments/src/components/Button/Button.types.tsx

```diff
@@ -15,12 +15,51 @@ export type IButtonStylesReturnType = ReturnType<Extract<IButtonComponent['style
 export type IButtonSlot = ISlotProp<IButtonProps>;
 
 export interface IButtonSlots {
+  /**
+   * Defines the root slot of the component.
+   */
   root?: IHTMLElementSlot<'button'>;
+
+  /**
+   * Defines the horizontal stack used for specifying inner layout of Button.
+   */
   stack?: IStackSlot;
+
+  /**
+   * Defines the text that is displayed inside the Button.
+   */
   content?: ITextSlot;
+
+  /**
+   * Defines the icon that is displayed next to the text inside the Button.
+   */
   icon?: IIconSlot;
+
+  /**
+   * Defines the contextual menu that appears when you click on the Button.
+   */
   menu?: IContextualMenuSlot;
+
+  /**
+   * Defines the menu chevron icon that is displayed insisde the Button.
+   */
   menuIcon?: IIconSlot;
+
+  // The following slots are specific to Split Button and aren't used if the split property is not present.
+  /**
+   * Defines the stack container for the primary action of the Split Button.
+   */
+  primaryActionContainer?: IStackSlot;
+
+  /**
+   * Defines the span container for the secondary action of the Split Button.
+   */
+  secondaryActionContainer?: IHTMLElementSlot<'span'>;
+
+  /**
+   * Defines the divider that separates the left and right parts of a Split Button.
+   */
+  splitDivider?: IHTMLElementSlot<'span'>;
```

Comment:
These could just be `IHTMLSlot`. It's a minor difference but I think it's a good one to make since Button will probably be a reference example. `IHTMLElementSlot` is needed for `button` primarily to allow access to button-specific attributes.